extensions/stackdriver/stackdriver.cc

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// NOLINT(namespace-envoy)
+#include <google/protobuf/util/json_util.h>
+#include <random>
+#include <string>
+#include <unordered_map>
+
+#include "stackdriver.h"
+
+#ifndef NULL_PLUGIN
+#include "api/wasm/cpp/proxy_wasm_intrinsics.h"
+#else
+
+#include "extensions/common/wasm/null/null.h"
+
+using namespace opencensus::exporters::stats;
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+namespace Null {
+namespace Plugin {
+#endif
+
+namespace Stackdriver {
+
+constexpr char kStackdriverExporter[] = "stackdriver_exporter";
+constexpr char kExporterRegistered[] = "registered";
+
+void StackdriverRootContext::onConfigure(
+    std::unique_ptr<WasmData> /* configuration */) {
```

Comment:
nit: give it a name in declaration, both here and in parent interface class.
And use below line to avoid warning
``` 
UNREFERENCED_PARAMETER(configuration);
```